modules/loader-utils/src/lib/library-utils/library-utils.js

```diff
@@ -5,7 +5,9 @@ import {global, isBrowser, isWorker} from '../env-utils/globals';
 import {requireFromFile, requireFromString} from './require-utils.node';
 import assert from '../env-utils/assert';
 
-const VERSION = typeof __VERSION__ !== 'undefined' ? __VERSION__ : 'latest';
+// TODO - unpkg.com doesn't seem to have a `latest` specifier for alpha releases...
+const LATEST = '2.0.0-alpha.3';
```

Comment:
`npm v @laoders.gl/core`

```
dist-tags:
1.1-latest: 1.1.7    beta: 2.0.0-alpha.3  latest: 1.3.3
```

So use `beta`?

https://unpkg.com/@loaders.gl/draco@beta/dist/libs/draco_wasm_wrapper.js

Reply:
Yes, it seems to work \o/ 

Nice find!  

Let's update after we land this, first we unbreak.